tools/RunGen.cpp

```diff
@@ -680,6 +801,10 @@ Usage: $NAME$ argument=value [argument=value... ] [flags]
         Override the default number of benchmarking iterations; ignored if 
         --benchmark is not also specified.
 
+    --benchmark_warmup=NUM: 
```

Comment:
Shouldn't really be necessary given that benchmarking already does multiple trials